scripts/contact.js

```diff
@@ -0,0 +1,43 @@
+function validation() {
+    var name = document.getElementById('name').value;
+    var num = document.getElementById('phone').value;
+    var mailid = document.getElementById('mail').value;
+
+    if(name == "") {
+        document.getElementById('name-error').innerHTML = "*Please fill the name field";
+        return false;
+    }
+    if((name.length <= 8) || (name.length > 25)) {
+        document.getElementById('name-error').innerHTML = "*Length must be between 8 and 25 characters";
+        return false;
+    }
+    if(num == "") {
+        document.getElementById('phone-error').innerHTML = "*Please fill the Phone number";
+        return false;
+    }
+    if(mailid == "") {
+        document.getElementById('email-error').innerHTML = "*Please fill the email id";
+        return false;
+    }
+}
+function loadSubmit() {
+    var name = document.getElementById('name').value;
+    var num = document.getElementById('phone').value;
+    var mailid = document.getElementById('mail').value;
+    let reqObj;
+    if(name && num && mailid) {
+        reqObj = {name, num, mailid};
+        console.log(reqObj);
+        reqObj = JSON.stringify(reqObj);
+        console.log(reqObj);
+        const xhrObj = new XMLHttpRequest();
```

Comment:
try to write a common utility function for XMLHttpRequest to use throughout the code